cg/cli/set.py

```diff
@@ -98,40 +121,84 @@ def samples(context, identifiers, kwargs, skip_lims, yes):
         )
 
 
+def list_changeable_sample_attributes(sample_obj: models.Sample = None, skip_list: list() = None):
+    """ list changeable attributes on sample """
+    keys = sample_obj.__dict__.keys() if sample_obj else models.Sample.__dict__.keys()
```

Comment:
Could this be made into a function to make it more readable and to better understand what it returns? For instance:
```
sample_attributes = get_sample_object_attributes()
```
I am also a bit confused about the use of keys and attributes? When should one use one over the other here?

Reply:
improved